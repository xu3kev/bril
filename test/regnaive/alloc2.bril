main {
  vj: int = const 1;
  vk: int = const 5;
  v1: int = const 1;
  v0: int = const 1;
  vg: int = add vj v1;
  vh: int = sub vk v0;
  vf: int = mul vg vh;
  v2: int = const 2;
  ve: int = add vj v2;
  v3: int = const 3;
  vm: int = add vj v3;
  vb: int = id vf;
  v4: int = const 5;
  vc: int = add ve v4;
  vd: int = id vc;
  vk: int = add vm v4;
  vj: int = id vb;
  print vj;
  print vk;
  print vd;
}
