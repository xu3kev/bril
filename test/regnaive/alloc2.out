8
9
8
#load: 15
#store: 26
